### DISCUSSION TITLE
Debug Area

### DISCUSSION THREAD

**Discussion by afranceschi-acv** [QuestPDF/QuestPDF#214] [Q&A]
Is it possible to have something like a Debug Area that includes the size of the rendered component? I'm picturing something where besides being able to specify the name to print we would be able to have the height x width in the same general format.

**Comment by MarcinZiabek** [comment]
This is quite interesting, thank you for the suggestion! Do you have more ideas what else could be useful, besides name and size? 😁

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
